fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -509,12 +509,29 @@
     @ConfField(mutable = true, masterOnly = true)
     public static int hadoop_load_default_timeout_second = 86400 * 3; // 3 day
 
+    // Configurations for spark load
+    /**
+     * Default spark dpp version
+     */
+    public static String spark_dpp_version = "1_0_0";
```

Comment:
```suggestion
    @ConfField
    public static String spark_dpp_version = "1_0_0";
```

All config must has `@ConfField` annotation.